CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 Notable updates:
 
 ### Added
+* [database] added ParityDb statistics to the stats endpoint [#2444](https://github.com/ChainSafe/forest/pull/2444).
+
 
 ### Changed
 * [libp2p] Ban peers with duration. Banned peers are automatically unbanned after a period of 1h.: [#2396](https://github.com/ChainSafe/forest/pull/2396)
+* [database] ParityDb fields `stats` and `compression` were renamed to `enable_statistics` and `compression_type` respectively. [#2444](https://github.com/ChainSafe/forest/pull/2444).
```

Comment:
What do you think of:

"* [config] `stats` and `compression` keys in `parity_db` section were renamed to `enable_statistics` and `compression_type` respectively."

Reply:
Sounds good